pkg/controller/endpointslice/reconciler.go

```diff
@@ -58,8 +58,8 @@ type reconciler struct {
 // endpointMeta includes the attributes we group slices on, this type helps with
 // that logic in reconciler
 type endpointMeta struct {
-	Ports       []discovery.EndpointPort `json:"ports" protobuf:"bytes,2,rep,name=ports"`
-	AddressType discovery.AddressType    `json:"addressType" protobuf:"bytes,3,rep,name=addressType"`
+	ports       []discovery.EndpointPort
+	addressType discovery.AddressType
```

Comment:
this seems to be here since the beginning https://github.com/kubernetes/kubernetes/commit/75f6c249235b40b24e9ea1efdb1ff81dd76a8d68

I would bet this is some leak from an internal prototype or something like that, @robscott may enlighten us

Reply:
Yeah I can't find any reason this should exist, likely accidentally leftover after copying.